### ISSUE TITLE
Relative positionning based on moving baseline?

### ISSUE THREAD

**Issue by AlexisTM** [tomojitakasu/RTKLIB#383]
Dear all,

I would like to compute the relative positioning between two GPS devices using V2V communication. 

Can I use RTKLib moving baseline algorithm to achieve it?

**Comment by DavidKelleySCSC** [comment]
Yes you can do that with RTKLIB, but in essence it does not scale well as you end up with a separate nav filter for each OBU device you care to track.  And the tracking periods can be very short for safety needs. In most data flows I have seen, the rover device broadcasts raw observation from itself device rather than (or along with) the BSM/CAM content.  As a concept, it been around about 20+ years, long before DSRC.  These days you here about it again with the advent of carrier data on the newer phones and the thought that a central repository can do the nav filter.  A thought that someone named Alison Brown promoted 30+ years ago.  The 'traditional' use of two devices is to get a better heading angle (think boat/plane) and that is what RTKLIB seems to have been designed for, I defer to Dr .Takasu in that. If you wish to PM me, about DSRC in this, feel free.  Success often comes from people who do not like to here 'no' abut an idea -so please do not take this as just negative.

**Comment by AlexisTM** [comment]
Thanks @DavidKelleySCSC for this useful information! 

I do see multiple applications in it inter robot interactions, like landing a copter on a vehicle. 

I am currently considering two solutions; Moving baseline between the two vehicles and RTK. Yet, the RTK requires to set up the base stations, which is more efficient if the areas of operations of multiple robots are close by; but it could be more efficient to compute locally the relative positions we require for the interaction.
